### PR TITLE
feat: align auth api with front convention

### PR DESCRIPTION
**Pull request by magrinj**
Align front auth with actual coding convention.
TBD in the front documentation

### CODE REVIEW

**Review by charlesBochet — APPROVED** [review]
Thank you!

File: front/src/modules/apollo/interfaces/apollo-manager.interface.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import { ApolloClient } from '@apollo/client';
```
Comment (charlesBochet) [inline front/src/modules/apollo/interfaces/apollo-manager.interface.ts:1]: apolloManager.interface.ts ?

File: front/src/modules/apollo/hooks/useApolloFactory.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { useEffect, useMemo, useRef } from 'react';
+import { InMemoryCache, NormalizedCacheObject } from '@apollo/client';
+import { useRecoilState } from 'recoil';
+
+import { tokenPairState } from '@/auth/states/tokenPairState';
+import { CommentThreadTarget } from '~/generated/graphql';
+
+import { ApolloFactory } from '../services/apollo.factory';
+
+export function useApolloFactory() {
```
Comment (lucasbordeau) [inline front/src/modules/apollo/hooks/useApolloFactory.ts:10]: Nice

File: front/src/modules/apollo/interfaces/apollo-manager.interface.ts (hunk 1 above)
Comment (magrinj) [reply front/src/modules/apollo/interfaces/apollo-manager.interface.ts]: Done :)

**Pull request merged**